src/field_screen_effect.c

```diff
@@ -1156,6 +1156,7 @@ static void Task_OrbEffect(u8 taskId)
         ClearGpuRegBits(REG_OFFSET_DISPCNT, DISPCNT_WIN1_ON);
         SetGpuRegBits(REG_OFFSET_BLDCNT, gOrbEffectBackgroundLayerFlags[0]);
         SetGpuReg(REG_OFFSET_BLDALPHA, BLDALPHA_BLEND(12, 7));
+        UpdateShadowColor(0x2109); // force shadows to gray
```

Comment:
Magic number

Reply:
Huh. This might be `RGB(9, 8, 8)`? Interesting that one of the components is different to the others.

Reply:
Yup, it is. Applied.